app/bundles/CoreBundle/Helper/ThemeHelper.php

```diff
@@ -95,20 +95,26 @@ class ThemeHelper
      * @var array
      */
     protected $defaultThemes = [
+        'Mauve',
         'aurora',
         'blank',
+        'brienz',
         'cards',
+        'coffee',
+        'confirmme',
         'fresh-center',
         'fresh-fixed',
         'fresh-left',
         'fresh-wide',
         'goldstar',
+        'nature',
         'neopolitan',
         'oxygen',
+        'paprika',
         'skyline',
         'sparse',
         'sunday',
-        'system',
```

Comment:
The `system` theme is actually very powerful feature. I tried to google for it but couldn't find any docs so it is hidden gem. You can overwrite any Mautic template from any bundle by this. For example if you copy `app/bundles/MauticEmailBundle/Views/Email/form.html.php` file into `themes/system/MauticEmailBundle/Email/form.html.php` then you can edit the email form template for that instance without hacking the core. This must stay.

Reply:
Wow ok, that's ... special :)

Looking at the source code, it's both the `system` theme and the `html` folder of the default / override theme are checked for the existence of the template file.

IMO this really needs to be documented (apart from this PR), as it's not clear what the possible use case of such theme is, or the html folder of any theme.